src/main/webapp/app/shared/sidebar/sidebar.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="d-flex sidebar rounded-3 px-3 sidebar-width" [ngClass]="{ 'sidebar-height-dev': !isProduction || isTestServer, collapsed: isCollapsed }">
+<div class="d-flex sidebar rounded-3 px-3 sidebar-width no-print" [ngClass]="{ 'sidebar-height-dev': !isProduction || isTestServer, collapsed: isCollapsed }">
```

Comment:
Maybe `hide-on-print` might be a better name? 

```suggestion
<div class="d-flex sidebar rounded-3 px-3 sidebar-width hide-on_print" [ngClass]="{ 'sidebar-height-dev': !isProduction || isTestServer, collapsed: isCollapsed }">
```

Reply:
done 👍